packages/wp-now/src/wp-now.ts

```diff
@@ -236,6 +237,16 @@ async function runPluginOrThemeMode(
 		projectPath,
 		`${documentRoot}/wp-content/${directoryName}/${pluginName}`
 	);
+	if (mode === WPNowMode.THEME) {
+		const templateName = getThemeTemplate(projectPath);
+		if (templateName) {
```

Comment:
Should we make sure the directory exists before mounting?

Reply:
Mount() does that now